### ISSUE TITLE
[Tooltip] Fix aria-labelledby attribute

### ISSUE THREAD

**Issue by ilyabrower** [semrush/intergalactic#922]
**Describe the bug**

The Tooltip should have the aria-labelledby  attribute but I can't find it.
Also Voice Over doesn't read the tooltip text when focused on trigger even if I manually set the aria-labelledby and id for the content.

**Expected behavior**
Tooltip has aria-labelledby  and Voice Over correctly read it.
